vm/src/cformat.rs

```diff
@@ -204,12 +204,18 @@ fn spec_format_string(
 fn try_update_quantity_from_element(
```

Comment:
Now quantity and precision has different spec, but they are handled by same function.
I don't think we necessarily need to split it to 2 functions, but adding a comment to `try_update_quantity_from_element` will be helpful to prevent confusion.

my suggestion is:
```suggestion
# This functions is used to parse both quantity and precision. Please consider to split this function once they doesn't fit in single function anymore.
fn try_update_quantity_from_element(
```

Reply:
Thanks for your feedback! Yeah, I think that would help ease the potential confusion.